arrow_pd_parser/export.py

```diff
@@ -67,3 +68,33 @@ def pd_to_json(
             new[col].replace("NaT", np.nan, regex=False, inplace=True)
 
     new.to_json(output_file, orient=orient, lines=lines, **kwargs)
+
+
+def pd_to_parquet(
+    df: pd.DataFrame,
+    output_file: Union[IO, str],
+    arrow_schema: pa.lib.Schema,
+):
+    """
+    Export a data frame as parquet
+    Does no conversion, and adheres to the provided schema
+
+    Args:
+        df (pd.DataFrame): a pandas dataframe
+        output_file (IO or str): the path you want to export to
+    """
+    if not isinstance(arrow_schema, pa.lib.schema):
+        raise TypeError(
+            f"arrow schema must be pyarrow schema, found {type(arrow_schema)}"
+        )
+    table = pa.Table.from_pandas(df)
+    table = table.cast(arrow_schema)
```

Comment:
The `from_pandas` function excepts a schema should probably be done there rather than casting afterwards?
https://arrow.apache.org/docs/python/generated/pyarrow.Table.html#pyarrow.Table.from_pandas